src/common/components/Auth/AuthCard.tsx

```diff
@@ -0,0 +1,18 @@
+import { type ComponentPropsWithoutRef, forwardRef } from "react";
+
+export const AuthCard = forwardRef<
+  HTMLDivElement,
+  ComponentPropsWithoutRef<"div">
+>(({ children, title = "Login" }, ref) => {
+  return (
+    <div
+      ref={ref}
+      className="mt-10 flex w-96 min-w-fit flex-col gap-y-5 rounded-lg bg-bg-alt px-4 py-5"
+    >
+      <div className="text-lg font-bold text-text-em-high">{title}</div>
```

Comment:
Let's use heading tag here for semantic HTML. Since this is the only header, I think you can use h1. 

Also, if possible, allow changing this heading tag via prop. Maybe can create and use some polymorphic type

Reply:
Thanks for the review Zachary! I've implemented polymorphism to the component, however I am unsure if there are ways to improve my current implementation. For example typescript doesn't give errors when I attempt to add additional props in `AuthCard` component on `login.tsx`. For example if I added `as={Link}` without a `href` argument, typescript doesn't show me that it requires a `href` in my editor. It only errors out on page load.